collector/process_collector.go

```diff
@@ -318,33 +319,34 @@ func (p *NamedProcessCollector) scrape(ch chan<- prometheus.Metric) {
 
 			if p.threads {
 				for _, thr := range gcounts.Threads {
+					threadName := filterNonutf8Characters(thr.Name)
```

Comment:
```suggestion
					threadName := strings.ToValidUTF8(thr.Name, "\uFFFD")
```

I believe this is the "replacement character" that's commonly used for invalid utf-8.